homeassistant/components/daikin/config_flow.py

```diff
@@ -130,6 +130,13 @@ async def async_step_zeroconf(self, discovery_info):
         """Prepare configuration for a discovered Daikin device."""
         _LOGGER.debug("Zeroconf user_input: %s", discovery_info)
         devices = Discovery().poll(ip=discovery_info[CONF_HOST])
+        if not devices:
```

Comment:
just curious what does the zeroconf send across data wise? any UUID or other idents?

Reply:
Nothing that can be used to uniquly identify the unit..

https://github.com/home-assistant/core/issues/37844#issuecomment-672822211

This have been discussed is several earlier PR.